spec/support/repository_shared.rb

```diff
@@ -1,39 +1,56 @@
 RSpec.shared_examples "a repository" do
 
   subject(:repository) { described_class.new(request) }
+
   let(:request) { instance_spy("Pipedrive::Request") }
 
   describe "#all" do
     it "returns all records from the repository" do
       repository.all
-      expect(request).to have_received(:get).with("/#{uri}")
+      expect(request).to have_received(:get)
     end
   end
 
-  describe "#where" do
-    it "returns records matching given field" do
-      search_data = {type: uri, field: "fake_field", value: "fake value"}
-      response = {id: 123, name: "Test"}
+  # personFields over-rides the behaviour of get_by_id so for that resource we
+  # need to skip these tests
```

Comment:
Yeah I think I'll change this so it's a parameter instead I wasn't happy with this